YAFC/Windows/WelcomeScreen.cs

```diff
@@ -1,4 +1,4 @@
-using System;
+﻿using System;
```

Comment:
Alright, `git diff head^ --word-diff-regex=.` finally showed something:
```
$ git diff head^ --word-diff-regex=.
diff --git a/YAFC/Workspace/ProjectPageView.cs b/YAFC/Workspace/ProjectPageView.cs
index 0d1e2cc..e04e823 100644
--- a/YAFC/Workspace/ProjectPageView.cs
+++ b/YAFC/Workspace/ProjectPageView.cs
@@ -1,4 +1,4 @@
{+﻿+}using System;
```

Reply:
It probably happens because we recently [added](https://github.com/have-fun-was-taken/yafc-ce/blob/master/.editorconfig#L15) [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) to `.editorconfig`.
Stackoverflow [says](https://stackoverflow.com/questions/49623731/what-is-the-default-encoding-for-source-files-in-visual-studio-2017) that Visual Studio detects a file by BOM. So that's probably why it wants the encoding to be `utf-8-bom`.

My assumption is that the files were previously saved by UTF-8 without BOM, so now it inserts that invisible mark at the start.

Reply:
I'll see what I can do.

Reply:
> It probably happens because we recently [added](https://github.com/have-fun-was-taken/yafc-ce/blob/master/.editorconfig#L15) [BOM](https://en.wikipedia.org/wiki/Byte_order_mark)

Yup, just tested it and confirmed it.

Reply:
[Done](https://github.com/have-fun-was-taken/yafc-ce/commit/94c01a9f97ebd6dbb6ec43af75f4c79c0ed1b982)